packages/xs/polly.0.4.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "polly"
+version: "0.4.1"
```

Comment:
There were quite a few changes to the C bindings here, I think there might still be some outstanding PRs on the upstream repo that I didn't review, I'll try to take another look.